apim-operator/deploy/analytics-configs/analytics_pv_template.yaml

```diff
@@ -18,7 +18,7 @@ kind: PersistentVolume
 apiVersion: v1
 metadata:
   name: analytics-pv-volume
-  namespace: wso2-system
+  namespace: default
```

Comment:
Why this is hard-corded ?

Reply:
removed this files, since we use emptydir volume to mount